generators/chipyard/src/main/scala/config/HeteroConfigs.scala

```diff
@@ -50,28 +46,6 @@ class HwachaLargeBoomAndHwachaRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithCoherentBusTopology ++
   new freechips.rocketchip.system.BaseConfig)
 // DOC include end: BoomAndRocketWithHwacha
-
-class DualLargeBoomAndRocketConfig extends Config(
```

Comment:
Why delete this config?

Reply:
The new `WithNCore` fragments compose more intuitively, such that it feels redundant to have all of these different configs. Heterogeneity is intuitive now.

Reply:
👍